scripts/generate/generate-src.ts

```diff
@@ -29,18 +56,20 @@ export function generateSrc(extensionType: string, template: Template) {
   const text = file.toString();
 
   try {
-    const outDir = path.resolve(__dirname, '../../src');
-    if (!fs.existsSync(outDir)) {
-      fs.mkdirSync(outDir);
+    if (!fs.existsSync(srcDir)) {
```

Comment:
They can't yet. I think when this was coded, the intention was to eventually expose it (just like we pre-emptively exposed the port for them to mod), but was taken out when we decided against having a config file. Although, I wouldn't mind leaving this logic for flexibility of changing our source path even if we never expose it to them. I can remove it though, if you'd like to wait for that time in the future to re-add this.

Reply:
I would prefer to remove it for now. The template is meant to work through the Shopify CLI and it looks like there's no way of setting this config through the CLI. It's being hardcoded to `rootDir` so having this as an option is not meaningful currently.

This one is slightly different than `build` and `serve` because the user can edit this configs outside of the CLI. They can't really do it for `init`

Reply:
I was assuming if they wanted they would do something like `shopify create extension --src=""` just like this future suggestion or adding port on `shopify serve --port=234` https://github.com/Shopify/app-extension-libs/issues/594

But any case, I'll just remove it for now.